integration/container/exec_test.go

```diff
@@ -102,6 +102,10 @@ func TestExec(t *testing.T) {
 	)
 	assert.NilError(t, err)
 
+	inspect, err := client.ContainerExecInspect(ctx, id.ID)
+	assert.NilError(t, err)
+	assert.Check(t, inspect.ExecID, id.ID)
```

Comment:
This is failing;

```
=== RUN   TestExec
--- FAIL: TestExec (1.08s)
    exec_test.go:107: invalid Comparison: 025a198fe3c944984814cc14a34bc3041cc46b12bb7917315409a90e65ee697b (string)
```

Should probably be

```suggestion
	assert.Check(t, inspect.ExecID == id.ID)
```

or

```suggestion
	assert.Check(t, is.Equal(inspect.ExecID, id.ID))
```

Reply:
🤦‍♂ 
Fixed